devicesimulation/single-vm/scripts/update.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -ex
+
+# Where to download scripts from
+REPOSITORY="https://raw.githubusercontent.com/Azure/pcs-cli/1.0.0-preview.3/remotemonitoring/single-vm"
+SCRIPTS_URL="${REPOSITORY}/scripts/"
+
+cd /app
+
+# Download scripts
+wget $SCRIPTS_URL/logs.sh     -O /app/logs.sh     && chmod 750 /app/logs.sh
+wget $SCRIPTS_URL/simulate.sh -O /app/simulate.sh && chmod 750 /app/simulate.sh
```

Comment:
this script should be removed from this solution, it won't work

Reply:
Removed